src/utility/defaultFonts.h

```diff
@@ -1,645 +1,937 @@
-    0x00, 0x3E, 0x41, 0x51, 0x21, 0x5E,             // Q
-    0x00, 0x7F, 0x09, 0x19, 0x29, 0x46,             // R
-    0x00, 0x46, 0x49, 0x49, 0x49, 0x31,             // S
-    0x00, 0x01, 0x01, 0x7F, 0x01, 0x01,             // T
-    0x00, 0x3F, 0x40, 0x40, 0x40, 0x3F,             // U
-    0x00, 0x1F, 0x20, 0x40, 0x20, 0x1F,             // V
-    0x00, 0x3F, 0x40, 0x38, 0x40, 0x3F,             // W
-    0x00, 0x63, 0x14, 0x08, 0x14, 0x63,             // X
-    0x00, 0x07, 0x08, 0x70, 0x08, 0x07,             // Y
-    0x00, 0x61, 0x51, 0x49, 0x45, 0x43,             // Z
-    0x00, 0x00, 0x7F, 0x41, 0x41, 0x00,             // [
-    0xAA, 0x55, 0xAA, 0x55, 0xAA, 0x55,             // обратный слеш
-    0x00, 0x00, 0x41, 0x41, 0x7F, 0x00,             // ]
-    0x00, 0x04, 0x02, 0x01, 0x02, 0x04,             // ^
-    0x00, 0x40, 0x40, 0x40, 0x40, 0x40,             // _
-
-    0x00, 0x00, 0x03, 0x05, 0x00, 0x00,             // `
-    0x00, 0x20, 0x54, 0x54, 0x54, 0x78,             // a
-    0x00, 0x7F, 0x28, 0x44, 0x44, 0x38,             // b
-    0x00, 0x38, 0x44, 0x44, 0x44, 0x20,             // c
-    0x00, 0x38, 0x44, 0x44, 0x48, 0x7F,             // d
-    0x00, 0x38, 0x54, 0x54, 0x54, 0x18,             // e
-    0x00, 0x08, 0x7E, 0x09, 0x01, 0x02,             // f
-    0x00, 0x18, 0xA4, 0xA4, 0xA4, 0x7C,             // g
-    0x00, 0x7F, 0x08, 0x04, 0x04, 0x78,             // h
-    0x00, 0x00, 0x44, 0x7D, 0x40, 0x00,             // i
-    0x00, 0x40, 0x80, 0x84, 0x7D, 0x00,             // j
-    0x00, 0x7F, 0x10, 0x28, 0x44, 0x00,             // k
-    0x00, 0x00, 0x41, 0x7F, 0x40, 0x00,             // l
-    0x00, 0x7C, 0x04, 0x18, 0x04, 0x78,             // m
-    0x00, 0x7C, 0x08, 0x04, 0x04, 0x78,             // n
-    0x00, 0x38, 0x44, 0x44, 0x44, 0x38,             // o
-
-    0x00, 0xFC, 0x24, 0x24, 0x24, 0x18,             // p
-    0x00, 0x18, 0x24, 0x24, 0x18, 0xFC,             // q
-    0x00, 0x7C, 0x08, 0x04, 0x04, 0x08,             // r
-    0x00, 0x48, 0x54, 0x54, 0x54, 0x20,             // s
-    0x00, 0x04, 0x3F, 0x44, 0x40, 0x20,             // t
-    0x00, 0x3C, 0x40, 0x40, 0x20, 0x7C,             // u
-    0x00, 0x1C, 0x20, 0x40, 0x20, 0x1C,             // v
-    0x00, 0x3C, 0x40, 0x30, 0x40, 0x3C,             // w
-    0x00, 0x44, 0x28, 0x10, 0x28, 0x44,             // x
-    0x00, 0x1C, 0xA0, 0xA0, 0xA0, 0x7C,             // y
-    0x00, 0x44, 0x64, 0x54, 0x4C, 0x44,             // z
-    0x00, 0x00, 0x10, 0x7C, 0x82, 0x00,             // {
-    0x00, 0x00, 0x00, 0xFF, 0x00, 0x00,             // |
-    0x00, 0x00, 0x82, 0x7C, 0x10, 0x00,             // }
-    0x00, 0x00, 0x06, 0x09, 0x09, 0x06,             // ~
-    0
+#ifndef __DEFAULT_FONTS_H__
+#define __DEFAULT_FONTS_H__
+
+#define fontRus6x8 alfabet6x8
+#define font6x8 alfabet6x8
+#define fontRus12x10 alfabet12x16
+#define font12x10 alfabet12x16
+#define mediumNumbers digits_7seg
+#define bigNumbers digits_14x24
+#define megaNumbers digits_24x40
+
```

Comment:
1. Names of define is uppercase letters.
`fontRus6x8`->`FONT_RUS_6x8`
2. I suggest using the same separator in all fonts: `space` or `_`
`digits_7seg`->`digits7seg`
`digits_14x24`->`digits14x24`
`digits_24x40`->`digits24x40`

Reply:
1 - NOT!!! I don`t create new constants. That '#defines' needed for access to old (outdated) font names with the new version of the library. Old font names were camel-case styled.
2 - OK. Will be camel-styled. Done